packages/patternfly-4/react-charts/src/components/ChartDonutUtilization/ChartDonutThreshold.tsx

```diff
@@ -426,65 +447,100 @@ export const ChartDonutThreshold: React.FunctionComponent<ChartDonutThresholdPro
     return Data.formatData(datum, { x, y, ...rest }, accessorTypes);
   };
 
-  // Returns the horizontal shift for the dynamic utilization donut cart
-  const getDynamicDonutDx = (dynamicTheme: ChartThemeDefinition, orientation: string) => {
+  // Returns the horizontal shift for the donut utilization chart
+  const getDonutDx = (dynamicTheme: ChartThemeDefinition, legendPosition: string) => {
     const dynamicWidth = donutWidth - (theme.pie.width - dynamicTheme.pie.width);
-    switch (orientation) {
-      case 'left':
-        return Math.round((donutWidth - dynamicWidth) / 2);
+    switch (legendPosition) {
       case 'right':
-        return -Math.round((donutWidth - dynamicWidth) / 2);
+        return Math.round((donutWidth - dynamicWidth) / 2);
       default:
         return 0;
     }
   };
 
-  // Returns the vertical shift for the dynamic utilization donut cart
-  const getDynamicDonutDy = (dynamicTheme: ChartThemeDefinition) => {
+  // Returns the vertical shift for the donut utilization chart
+  const getDonutDy = (dynamicTheme: ChartThemeDefinition) => {
     const dynamicHeight = donutHeight - (theme.pie.height - dynamicTheme.pie.height);
     return Math.round((donutHeight - dynamicHeight) / 2);
   }
 
+  // Returns the vertical shift for the donut utilization legend and subtitle
+  const getDy = (dynamicTheme: ChartThemeDefinition, position: string) => {
```

Comment:
```suggestion
  const getLegendDy = (dynamicTheme: ChartThemeDefinition, position: string) => {
```

Reply:
I'm using this to shift both legend and subtitle, so I had renamed getLegendDy as getDy

Reply:
I'll rename it as getLegendAndSubTitleDy to be more clear

Reply:
Oh, whoops. Sorry, missed that in the comment